packages/fxa-content-server/scripts/test-ci-remote.sh

```diff
@@ -43,9 +43,6 @@ function test_suite() {
     --groupIndex=${i}
 }
 
-# TODO: Move to step in CI
```

Comment:
Not why this run at this point. Removing it. We now have a lint job that takes care of this.